orchestrator/packages/db/src/scripts/delete-api-key.ts

```diff
@@ -0,0 +1,30 @@
+import { parseArgs } from "util";
+import prismaInstance from "../prisma-instance";
+
+const main = async () => {
+  const { values } = parseArgs(
+    {
+      options:
+      {
+        hostname: { type: "string", short: "h" },
+        key: { type: "string", short: "k" }
+      },
+    }
+  );
+  const missing = Object.entries(values).filter(([_, v]) => v === undefined).map(([k, _]) => k);
```

Comment:
This would be mildly more convenient to use if the -k option is optional when the -h host has a single unique key...  Not urgent, but more convenient.